src/pages/gen2/build-a-backend/functions/additional-configuration/index.mdx

```diff
@@ -0,0 +1,81 @@
+{/* // import { getChildPageNodes } from '@/utils/getChildPageNodes'; */}
+
+export const meta = {
+  title: 'Additional configuration',
```

Comment:
Resist titles like "additional" or "advanced" they become catch all buckets with limited meaning for the customer.